Source/WebKit/NetworkProcess/webrtc/NetworkRTCUDPSocketCocoa.mm

```diff
@@ -233,7 +233,7 @@ static inline bool isNat64IPAddress(const webrtc::IPAddress& ip)
     configureParameters(parameters.get(), address.family() == AF_INET ? nw_ip_version_4 : nw_ip_version_6);
 
     m_nwListener = adoptNS(nw_listener_create(parameters.get()));
-    nw_listener_set_queue(m_nwListener.get(), udpSocketQueue());
+    nw_listener_set_queue(m_nwListener.get(), RetainPtr { udpSocketQueue() }.get() );
```

Comment:
ditto about renaming instead.

Reply:
In my new PR I am not touching this file. Good to see so many of NetworkProcess warnings fixed now!